### PR TITLE
docs: fine-grained tokens

### PR DESCRIPTION
**Pull request by rephus**
Closes: #3866

### Description:

https://deploy-preview-3876--peaceful-bassi-cbf284.netlify.app/get-started/setup-lightdash/connect-project#fine-grained-access-token-beta

image

### CODE REVIEW

File: docs/docs/get-started/setup-lightdash/connect-project.mdx [hunk 1]
```diff
@@ -439,6 +446,31 @@ Your personal access token can be found in your user settings in databricks:
 
 <img src={DatabricksAccessScreenshot} width="1097" height="647" style={{display: "block", margin: "0 auto 20px auto"}}/>
 
+##### Fine-grained access token (beta) 
+
+Fine-grained access tokens are new special tokens that can only give access to individual repositories on your github account. 
+You can read more about on the [Github docs](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-fine-grained-personal-access-token).
```
Comment (TuringLovesDeathMetal) [inline docs/docs/get-started/setup-lightdash/connect-project.mdx:452]:
```suggestion
You can read more about it on the [Github docs](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-fine-grained-personal-access-token).
```

File: docs/docs/get-started/setup-lightdash/connect-project.mdx [hunk 2]
```diff
@@ -439,6 +446,31 @@ Your personal access token can be found in your user settings in databricks:
...
+
+1. Go to [settings > Developer access > Personal access tokens > fine-grained token](https://github.com/settings/personal-access-tokens/new)
+2. Add the name, expiration, description and owner (we'll need his username later)
```
Comment (TuringLovesDeathMetal) [inline docs/docs/get-started/setup-lightdash/connect-project.mdx:455]:
```suggestion
2. Add the name, expiration, description and owner (we'll need their username later)
```

File: docs/docs/get-started/setup-lightdash/connect-project.mdx [hunk 3]
```diff
@@ -431,6 +434,10 @@ Follow the instructions above to find your ODBC connection instructions.
 
 #### Personal Access Token
 
+You can create a personal access token (classic), or a fine-grained access token (beta).
```
Comment (TuringLovesDeathMetal) [inline docs/docs/get-started/setup-lightdash/connect-project.mdx:437]: I think that you added this in the Databricks bit! Not the GitHub bit :)

**Comment by TuringLovesDeathMetal** [comment]
@rephus I think you added this under the `Databricks` part of the docs. 

Right now, we just link people to the GitHub docs:

image


Maybe we could just do the same now, but add a section below which is like:

If you've used the classic personal access token creation, then enter your personal access token like this: 
my_personal_access_token

If you've used the fine-grained personal access token, then enter your personal access token like this:
github_username:token
image

**Comment by rephus** [comment]
thanks @TuringLovesDeathMetal I fixed the databricks section, this is how it looks now https://deploy-preview-3876--peaceful-bassi-cbf284.netlify.app/get-started/setup-lightdash/connect-project/#github Do you still prefer to keep it simple and only point to the docs ? (we should also mention to give `contents`  permission)

**Comment by TuringLovesDeathMetal** [comment]
@rephus this looks great!

**Comment by rephus** [comment]
New e2e headless chrome tests are failing... ignoring for now

**Pull request merged**